### DISCUSSION TITLE
Add A Macro Command Enry and Processing.

### DISCUSSION THREAD

**Discussion by jgbrown54** [Guizz27/DWIN_LK4Pro#12] [Ideas]
Allow a user to store a sequence of gcodes and name them. And provide a means to locate and launch a saved sequence.

**Comment by Guizz27** [comment]
Sounds a little bit like Marlin gcode macros https://marlinfw.org/docs/gcode/M810-M819.html.

**Comment by jgbrown54** [comment]
That would provide the functionality. I thought it might be nice to enter them from the display and give them a name and a way to list them and select one to run. To be honest, I'm not sure what I would use it for. I just thought it might be nice. Definately not a priority until someone comes up with a legimate use. I could do the same thing with a gcode file on the sd card with a name.
